src/index.js

```diff
@@ -0,0 +1,152 @@
+import './style.css';
+import './popup.css';
+import './media.css';
+import { temp, renderPopUp } from './modules/popup.js';
+import getShow from './modules/get-show.js';
+import displayComment from './modules/commentTemp.js';
+import { updateLikes } from './modules/save-get-likes.js';
+
+const card = document.querySelector('.cards');
+const numberItem = document.querySelector('.number');
+const acceil = document.querySelector('.title');
+const searchArea = document.querySelector('.Search');
+const homecard = document.querySelector('.home_cards');
+const displayhomeItem = (result) => {
+  homecard.innerHTML = '';
+  result.forEach((item) => {
+    const scoreLi = document.createElement('div');
+    scoreLi.className = 'card';
+    scoreLi.innerHTML = `<img src="${item.image.medium}">
+                         <p class="movi-title">${item.name}</p>
+                         <div class='buttons-holder'>
+                         <button type="submit" class="comment-btn">Comment</button>
+                         <div class='likes-holder'>
+                         <i class="fa-regular fa-heart"></i><span class='likes-num'>0</span>
+                         </div>
+                         </div>`;
+    homecard.appendChild(scoreLi);
+  });
+  const cards = document.querySelectorAll('.card');
+  numberItem.textContent = cards.length;
+};
+
+const displayItem = (results) => {
+  card.innerHTML = '';
+  results.forEach((item) => {
+    const scoreLi = document.createElement('div');
+    scoreLi.className = 'card';
+    scoreLi.innerHTML = `<img src="${item.show.image.medium}">
+                         <p class="movi-title">${item.show.name}</p>
+                         <div class='buttons-holder'>
+                         <button type="submit" class="comment-btn">Comment</button>
+                         <div class='likes-holder'>
+                          <i class="fa-regular fa-heart"></i><span class='likes-num'>0</span>
+                          </div>
+                         </div>`;
+    card.appendChild(scoreLi);
+  });
+  const cards = document.querySelectorAll('.card');
+  numberItem.textContent = cards.length;
+};
+
+const backgroundBlur = () => {
+  const popup = document.querySelector('.popup');
+  const cover = document.querySelector('.cover');
+  const body = document.querySelector('body');
+  cover.classList.add('d-block');
+  body.style.overflow = 'hidden';
+  const closeBtn = document.querySelector('.fa-xmark');
+  closeBtn.addEventListener('click', () => {
+    popup.classList.add('d-none');
+    cover.classList.remove('d-block');
+    body.style.overflowY = 'scroll';
+  });
+};
+
+const displayPopUp = async (id) => {
+  const popup = document.querySelector('.popup');
+  await getShow(id).then((data) => {
+    popup.innerHTML = '';
+    popup.classList.remove('d-none');
+    popup.style.width = '80%';
+    popup.style.zIndex = '200';
+    renderPopUp(temp(data));
+    backgroundBlur();
+    displayComment(id);
+  });
+};
```

Comment:
- [ ] Remember that you are to use the `async` and `await` with `fetch`. Just like the other reviewer said, using `async` and `await` with the promise `thenable chain` is not the best practice. You would want to either use `async` and `await` or the `thenable` chain for this project. One of the major requirements for this project is that you use async and await. Kindly implement this change.